Entities/Common/Emotes/EmoteHotkeys.as

```diff
@@ -31,36 +11,15 @@ void onInit(CBlob@ this)
 
 	this.addCommandID("prevent emotes");
 
-	//attempt to load from cache first
-	ConfigFile@ cfg = openEmoteBindingsConfig();
-
-	emote_1 = read_emote(cfg, "emote_1", Emotes::attn);
-	emote_2 = read_emote(cfg, "emote_2", Emotes::smile);
-	emote_3 = read_emote(cfg, "emote_3", Emotes::frown);
-	emote_4 = read_emote(cfg, "emote_4", Emotes::mad);
-	emote_5 = read_emote(cfg, "emote_5", Emotes::laugh);
-	emote_6 = read_emote(cfg, "emote_6", Emotes::wat);
-	emote_7 = read_emote(cfg, "emote_7", Emotes::troll);
-	emote_8 = read_emote(cfg, "emote_8", Emotes::disappoint);
-	emote_9 = read_emote(cfg, "emote_9", Emotes::ladder);
-
-	emote_10 = read_emote(cfg, "emote_10", Emotes::flex);
-	emote_11 = read_emote(cfg, "emote_11", Emotes::down);
-	emote_12 = read_emote(cfg, "emote_12", Emotes::smug);
-	emote_13 = read_emote(cfg, "emote_13", Emotes::left);
-	emote_14 = read_emote(cfg, "emote_14", Emotes::okhand);
-	emote_15 = read_emote(cfg, "emote_15", Emotes::right);
-	emote_16 = read_emote(cfg, "emote_16", Emotes::thumbsup);
-	emote_17 = read_emote(cfg, "emote_17", Emotes::up);
-	emote_18 = read_emote(cfg, "emote_18", Emotes::thumbsdown);
-
+	CPlayer@ me = getLocalPlayer();
+	emoteBinds = readEmoteBindings(me);
```

Comment:
Added the null check to prevent this error. However, I would hope that `me` is never null here so the emote bindings are correctly loaded.

If you could find reproduction steps that would be great, otherwise the null check would have to do for now.

Reply:
Dedicated servers?